Geometry/TrackerCommonData/data/PhaseII/TiltedTracker404/pixbar.xml

```diff
@@ -151,22 +151,19 @@ note: see OT614_200_IT404_2017_12_12.cfg for full config files
     <Constant name="RadialCoolZ" value="29.5*cm"/>
     <Constant name="RadialCoolDZ" value="1.0*cm"/>
     <Constant name="RadialCoolRin" value="4.0*cm"/>
+    <Constant name="Tracker2SOuterRadius" value="116.0*cm"/>
```

Comment:
Tracker version 404 is already using in existing geometry D21, so shouldn't be changed. These changes are causing 1000s of differences in the D21 comparisons.

Reply:
see my discussion below, this is not changing the layour, while we have already changed it a few weeks ago